src/NuGet.Core/NuGet.Packaging.Extraction/Strings.resx

```diff
@@ -138,10 +138,12 @@
   <data name="ErrorNullOrEmptyPackageId" xml:space="preserve">
     <value>Null or empty package id</value>
   </data>
-  <data name="Log_InstallingPackage" xml:space="preserve">
-    <value>Installing {0} {1}.</value>
+  <data name="Log_InstalledPackage" xml:space="preserve">
```

Comment:
I know there's been a chat about this, can you please remind why we have to duplicate this file and instead just embed it?

Reply:
When I linked `Strings.resx`, but also keeping the `Generator` metadata, the resx code generator creates a designer.cs file where the namespace is `NuGet.Packaging.Extraction...NuGet.Packaging`, which fails to compile. When I don't set the generator, and compare the binlog, I see `Strings.resx`'s `OutputResource` metadata is a little different, and therefore the command line arguments for `csc.exe` are a little different. It looks like it should work, it certainly compiles. But I'm not sure it'll actually work at runtime, and I'm not willing to spend more time investigating.